<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Init_Coder</class>
    <widget class="QWidget" name="Init_Coder">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>288</width>
                <height>229</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>0</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>16777215</width>
                <height>16777215</height>
            </size>
        </property>
        <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops">
            <bool>true</bool>
        </property>
        <property name="windowTitle">
            <string>Создание кодера</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../../../Resources/img/img.qrc">
                <normaloff>:/MainWindow/AddCoder.png</normaloff>:/MainWindow/AddCoder.png
            </iconset>
        </property>
        <property name="locale">
            <locale language="Russian" country="Russia"/>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>20</y>
                    <width>271</width>
                    <height>52</height>
                </rect>
            </property>
            <layout class="QGridLayout" name="grid">
                <property name="spacing">
                    <number>10</number>
                </property>
                <item row="1" column="1">
                    <widget class="QPushButton" name="submit_button">
                        <property name="enabled">
                            <bool>false</bool>
                        </property>
                        <property name="focusPolicy">
                            <enum>Qt::NoFocus</enum>
                        </property>
                        <property name="text">
                            <string>Подтвердить</string>
                        </property>
                        <property name="shortcut">
                            <string>Return</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="0">
                    <widget class="QComboBox" name="choose_comboBox">
                        <property name="contextMenuPolicy">
                            <enum>Qt::ActionsContextMenu</enum>
                        </property>
                        <property name="currentText">
                            <string>Выбор кодера</string>
                        </property>
                        <item>
                            <property name="text">
                                <string>Выбор кодера</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>Хемминга</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>Циклический</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>Сверточный</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>Фонтанный</string>
                            </property>
                        </item>
                    </widget>
                </item>
                <item row="2" column="0">
                    <widget class="QRadioButton" name="first_coder_radio_button">
                        <property name="text">
                            <string>Первый кодер</string>
                        </property>
                        <property name="checked">
                            <bool>true</bool>
                        </property>
                    </widget>
                </item>
                <item row="2" column="1">
                    <widget class="QRadioButton" name="second_coder_radio_button">
                        <property name="text">
                            <string>Второй кодер</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget_2">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>90</y>
                    <width>271</width>
                    <height>131</height>
                </rect>
            </property>
            <layout class="QGridLayout" name="grid_coder_options">
                <property name="spacing">
                    <number>10</number>
                </property>
            </layout>
        </widget>
    </widget>
    <resources>
        <include location="../../../Resources/img/img.qrc"/>
    </resources>
    <connections/>
</ui>
